### PR TITLE
fix for modern browser build with CSP

### PR DESCRIPTION
**Pull request by ameinhardt**
remove new Function and undefined variable

### CODE REVIEW

File: src/tables/GPOS.js [hunk 1]
```diff
@@ -185,7 +185,7 @@ let GPOSLookup = new r.VersionedStruct('lookupType', {
   9: { // Extension Positioning
     posFormat:   r.uint16,
     lookupType:  r.uint16,   // cannot also be 9
-    extension:   new r.Pointer(r.uint32, GPOSLookup)
```
Comment (Pomax) [inline src/tables/GPOS.js]: Hmm, this doesn't look like an eval/new Function removal.

File: src/tables/GSUB.js [hunk 2]
```diff
@@ -53,7 +53,7 @@ let GSUBLookup = new r.VersionedStruct('lookupType', {
   7: { // Extension Substitution
     substFormat:   r.uint16,
     lookupType:    r.uint16,   // cannot also be 7
-    extension:     new r.Pointer(r.uint32, GSUBLookup)
```
Comment (Pomax) [inline src/tables/GSUB.js]: This also doesn look like an eval/new Function removal.

File: src/tables/GPOS.js (hunk 1 above)
Comment (ameinhardt) [reply src/tables/GPOS.js]: @Pomax: true. But it's a fix for modern browser that honor 'let'. The varible GPOSLookup is not defined at that point at time anyway. If converted to 'var' it is undefined.

File: src/tables/GSUB.js (hunk 2 above)
Comment (ameinhardt) [reply src/tables/GSUB.js]: @Pomax: true. But it's a fix for modern browser that honor 'let'. The varible GSUBLookup is not defined at that point at time anyway. If converted to 'var' it is undefined.

File: src/tables/GPOS.js (hunk 1 above)
Comment (Pomax) [reply src/tables/GPOS.js]: I would strongly recommend filing a separate PR for that. As a PR to fix a CSP problem, this line (and the one for GSUB) are not part of the issue being fixed, and deserve their own issue, with their own PR.

**Comment by Pomax** [comment]
@devongovett or @ameinhardt can you close this PR? It's super out of date and I think it's safe to say no one's going to be revisiting it, but while it's open it keeps showing up in https://github.com/pulls/mentioned

**Pull request closed without merging**